src/snapshotter.test.js

```diff
@@ -124,4 +125,19 @@ describe('The snapshotter', () => {
     await mockSnapshot.takeSnap();
     expect(mockSnapshot.driver.addCookie.mock.calls.length).toBe(2);
   });
+
+  it('Executes a script', async () => {
```

Comment:
you're testing two things here and I think there should be two test cases although the logic is identical.

if things went south this would be a pain to debug

Reply:
okies will separate this to two tests